app/controllers/plans_controller.rb

```diff
@@ -330,6 +332,52 @@ def set_test
   end
 
 
+  # Collect all of the templates available for the org+funder combination
+  # --------------------------------------------------------------------------
+  def template_options()
```

Comment:
Have you considered moving this logic to the templates model?

Reply:
yes, that would be more appropriate